<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QWidget" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginView</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>140</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QToolButton" name="userBtn">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>812</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/user_static.png);}
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/user_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/user_mouse_click.png); }
QToolButton::menu-indicator{	image: none;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>844</y>
      <width>60</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="menuBtn">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>882</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/menu_static.png); }
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/menu_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/menu_mouse_click.png) ;}
QToolButton::menu-indicator{	image: none;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resource/resource.qrc">:/svpk_lib/images/sovajo_logo/logo_cn_middle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="screenshotBtn">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>932</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/screenshot_static.png); }
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/screenshot_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/screenshot_mouse_click.png) ;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="shutdownBtn">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>982</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/shutdown_static.png) ;}
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/shutdown_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/shutdown_mouse_click.png); }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetRight" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>1780</width>
     <height>1080</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
